### ISSUE TITLE
instructions for building app

### ISSUE THREAD

**Issue by fouzan14** [AbdulMalikDev/ounotes#39]
Hi,

I would like to build this app from null_safety branch and since I'm new to flutter could you please provide build instructions.

**Comment by Ankitkj1999** [comment]
Yes, I also need a build instruction guide. I have been trying to make it run on my device for more than a week, but all my efforts are in vain.

**Comment by buildwithmalik** [comment]
Bit busy with final sem exams. I intend to write a guide on how to setup next month. Please remind me if I've forgotten to do so.

**Comment by zahqresh** [comment]
Hey Malik, hope you are doing fine. When do you intend to write the guide?

**Comment by Im2rnado** [comment]
Need instructions too please

**Comment by Ankitkj1999** [comment]
Please see to it!!

**Comment by UltraDeep** [comment]
Please

> Bit busy with final sem exams. I intend to write a guide on how to setup next month. Please remind me if I've forgotten to do so.

Reminder: It's been over a month now.

**Comment by buildwithmalik** [comment]
Yep thanks for the reminders but I think writing a guide will take a long time. I'm just thinking of making a video explaining how to build the app.

What do you guys think about this idea?

**Comment by UltraDeep** [comment]
I think it is a good idea as long as it is clear. Because I can't even get it to run on my machine. So clear instructions on how to build it and setup the backend will be highly appreciated on my side.
😉

**Comment by Ankitkj1999** [comment]
Yes, a video tutorial would be great!

**Comment by Ankitkj1999** [comment]
Please give it some time!

**Comment by UltraDeep** [comment]
> Yep thanks for the reminders but I think writing a guide will take a long time. I'm just thinking of making a video explaining how to build the app.
> 
> What do you guys think about this idea?

Please don't forget about the video
